src/main/frontend/model/samples/value-finder.json

```
@@ -0,0 +1,76 @@
+{
+	"logLevel": "info",
+	"hops": [
+		{ "type": "declare", "declarations": { "output": "utils.file.txt(\"properties_containing_hardcoded_urls\")" } },
```

I think there’s a namespace for `file`:
```suggestion
		{ "type": "declare", "declarations": { "output": "file:txt(\"properties_containing_hardcoded_urls\")" } },
```